code/modules/surgery/organs/tongue.dm

```diff
@@ -318,3 +318,51 @@
 /obj/item/organ/tongue/ethereal/Initialize(mapload)
 	. = ..()
 	languages_possible = languages_possible_ethereal
+
+//Sign Language Tongue - yep, that's how you speak sign language.
+/obj/item/organ/tongue/tied
+	name = "tied tongue"
+	desc = "If only one had a sword so we may finally untie this knot."
+	say_mod = "signs"
+	icon_state = "tonguetied"
+	modifies_speech = TRUE
+
+/obj/item/organ/tongue/tied/Insert(mob/living/carbon/M)
+	. = ..()
+	M.verb_ask = "signs"
+	M.verb_exclaim = "signs"
+	M.verb_whisper = "subtly signs"
+	M.verb_sing = "rythmically signs"
+	M.verb_yell = "emphatically signs"
+	ADD_TRAIT(M, TRAIT_SIGN_LANG, "tongue")
+	REMOVE_TRAIT(M, TRAIT_MUTE, "tongue")
+
+/obj/item/organ/tongue/tied/Remove(mob/living/carbon/M, special = 0)
+	..()
+	M.verb_ask = "asks"
+	M.verb_exclaim = "exclaims"
+	M.verb_whisper = "whispers"
+	M.verb_sing = "sings"
+	M.verb_yell = "yells"
```

Comment:
verb_ask could be overriden or changed in the future right? all these could be.

Use initial() to assign them to their initial value to prevent this
http://www.byond.com/docs/ref/#/proc/initial